modules/monitoring-specifying-how-a-service-is-monitored.adoc

```diff
@@ -29,30 +28,40 @@ The `prometheus-example-app` sample service does not support TLS authentication.
 
 .Procedure
 
-. Create a YAML file for the `ServiceMonitor` resource configuration. In this example, the file is called `example-app-service-monitor.yaml`.
+. Create a new YAML configuration file named `example-app-service-monitor.yaml`.
 
-. Add the following `ServiceMonitor` resource configuration details:
+. Add a `ServiceMonitor` resource to the YAML file. The following example creates a service monitor named `prometheus-example-monitor`:
 +
 [source,yaml]
 ----
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  labels:
-    k8s-app: prometheus-example-monitor
   name: prometheus-example-monitor
-  namespace: ns1
+  namespace: ns1 #<1>
 spec:
-  endpoints:
-  - interval: 30s
+  endpoints: #<2>
+  - authorization:
+      credentials:
+        key: token
+        name: example
+    interval: 30s 
     port: web
     scheme: http
-  selector:
+  selector: #<3>
     matchLabels:
       app: prometheus-example-app
 ----
+<1> Specify a user-defined namespace where you want the `ServiceMonitor` resource to run.
+<2> Configure an endpoint for metrics to be scraped by Prometheus.
+<3> Configure selector to specify the endpoints objects. This example scrapes the metrics exposed by the `prometheus-example-app` service.
 +
-This defines a `ServiceMonitor` resource that scrapes the metrics exposed by the `prometheus-example-app` sample service, which includes the `version` metric.
+[IMPORTANT]
+====
+If you want to configure a bearer token for scraping the target, use an `authorization` object instead of the deprecated `bearerTokenFile` and `bearerTokenSecret`.
```

Comment:
NOTE: Because `bearerTokenFile` and `bearerTokenSecret` are not deprecated for OCP 4.12 and 4.13, I am planning to remove this first sentence of this admonition while cherry-picking in the 4.12 and 4.13 versions. The second sentence will stay.

Reply:
I might just specify it more by saying:
```
If you use the `bearerTokenFile` object to configure a bearer token for scraping the target, the `ServiceMonitor` resource is rejected. Use an `authorization` object instead.
```

Reply:
I don't thinks we need to change now, see from https://issues.redhat.com/browse/OCPBUGS-31847, we only see the warnings under debug loglevel now, and servicemonitor with `bearerTokenFile` is  Deprecated, not rejected
```
$ oc explain servicemonitor.spec.endpoints.bearerTokenFile
GROUP:      monitoring.coreos.com
KIND:       ServiceMonitor
VERSION:    v1

FIELD: bearerTokenFile <string>

DESCRIPTION:
    File to read bearer token for scraping the target. 
     Deprecated: use `authorization` instead.
```

Reply:
Hello @simonpasquier I wanted to check with you before we close this one. WDYT?